app/models/service_provider.rb

```diff
@@ -47,6 +47,14 @@ def encryption_opts
     }
   end
 
+  def skip_encryption_allowed
+    config = AppConfig.env.skip_encryption_allowed_list
+    return false unless config
+    
+    @allowed_list ||= JSON.parse(config, symbolize_names: true)
```

Comment:
since the value here is an array instead of an object, the `symbolize_names` doesn't really do anything? We could remove
```suggestion
    @allowed_list ||= JSON.parse(config)
```